_posts/courses/earth-analytics/07-multispectral-remote-sensing/2017-01-01-week-07-multispectral-remote-sensing-landsat.md

```diff
@@ -33,7 +33,7 @@ At the end of this week you will be able to:
 * Use the Landsat file naming convention to determine correct band combinations for plotting and calculating NDVI.
 * Define additive color model.
 
-{% include /data_subsets/course_earth_analytics/_data-week6-7.md %}
+{% include/data_subsets/course_earth_analytics/_data-week6-7.md %}
```

Comment:
@jlpalomino did this actually build locally for you? normally an include needs a space after the word "include"

Reply:
@lwasser I noticed that too in all of the files, but yes, it did build locally for me.

Reply:
@lwasser I should clarify - that is not something that I changed but was modified during the build.

Reply:
oh that's really weird. let me look at that more closely. i suppose if it builds it's ok. but weird knitr would do that.

Reply:
ok i see ths space is missing here too:
https://github.com/earthlab/earth-analytics-lessons/pull/301/files#diff-04042eb84997768291c8b32d9abade23R36 
but it looks like it wasn't a change - just missing a space before and after your edits. 

can you please fix the space and run it through the build again and see if it remains in tact? thank you
just want to ensure the build isn't doing something weird

Reply:
the good news is you can run it through the same branch as that space will be a commit change that will trigger a push to the website repo!!

Reply:
@lwasser I added the space and successfully rebuilt the week 7 lessons. The md files retained the space and the pages build locally.  So confirmed that the build is not modifying that spacing; the space was missing before because it was missing in the original Rmd files for week 7.